examples/step-61/doc/intro.dox

```diff
@@ -0,0 +1,280 @@
+FE_FaceQ as the face polynomial space, and Raviart-Thomas elements for the velocity. 
+For the numerical velocity $\mathbf{u} = -{\mathbf{K}} \nabla_{w,d} p$, 
+it is defined on the vector space, Raviart-Thomas spaces $RT(k)$.
+
+<h3> Assembling the linear system </h3>
+
+First, we solve for the pressure. 
+We collect two local spaces together in one FESystem, 
+the first component in this finite element system denotes 
+the space for interior pressure, and the second denotes 
+the space for face pressure. 
+For the interior component, we use the polynomial space FE_DGQ. 
+For the face component, we use FE_FaceQ.
+
+We use shape functions defined on spaces FE_DGQ and FE_FaceQ to 
+approximate pressures, i.e., $p_h = \sum a_i \phi_i,$ 
+where $\phi_i$ are shape functions of FESystem. 
+We construct the local system by using discrete weak gradients of 
+shape functions of FE_DGQ and FE_FaceQ.
+The discrete weak gradients of shape functions $\nabla_{w,d} \phi$ are defined as 
+$\nabla_{w,d} \phi = \sum_{i=1}^m c_i \mathbf{w}_i,$ 
+where $\mathbf{w}_i$ is the basis function of $RT(k)$, 
+$m$ is the dof of the $RT(k)$. 
+
+Using integration by parts, we have a small linear system 
+on each element $K$,
+@f{equation*}
+\int_{E} \left(\nabla_{w,d} \phi \right) \cdot \mathbf{w} = 
+\int_{E^\partial} \phi^{\partial} \left(\mathbf{w} \cdot \mathbf{n}\right) - 
+\int_{E^\circ} \phi^{\circ} \left(\nabla \cdot \mathbf{w}\right),  
+\quad \forall \mathbf{w} \in RT_{[k]}(E),
+@f}
+
+@f{equation*}
+\sum_{i=1}^m c_i \int_E \mathbf{w}_i \cdot \mathbf{w}_j = 
+\int_{E^{\partial}} \phi_i^{\partial} 
+\left(\mathbf{w}_j \cdot \mathbf{n} \right) - 
+\int_{E^{\circ}} \phi_i^{\circ} \left (\nabla \cdot \mathbf{w}_j \right),
+@f}
+which can be simplified to be
+@f{equation*}
+\mathbf{C}_{E}\mathbf{M}_{E} = \mathbf{F}_{E},
+@f}
+where $\mathbf{C}_E$ is the matrix with unknown coefficients $c$, 
+$\mathbf{M}_E$ is the Gram matrix 
+$\left[  \int_E \mathbf{w}_i \cdot \mathbf{w}_j \right]$, 
+$\mathbf{F}_E$ is the matrix of right hand side, 
+$\mathbf{w}$ and $\phi_i^{\circ}$ are in FEValues, 
+$\phi_i^{\partial}$ is in FEFaceValues. 
+Then we solve for $\mathbf{C}_E = \mathbf{F}_E \mathbf{M}_E^{-1}$. 
+Now, discrete weak gradients of shape functions are written as 
+linear combinations of basis functions of the $RT$ space. 
+In our code, we name $\mathbf{C}_E$ as \textit{cell\_matrix\_C}, 
+$\mathbf{M}_E$ as <code>cell_matrix_rt</code>, 
+$\mathbf{F}_E$ as <code>cell_matrix_F</code>.
+
+On the local matrix $\mathbf{A}$ of the element, each component is 
+@f{equation*}
+\mathbf{A}_{ij} = 
+\int_{E} \mathbf{K} \nabla_{w,d} \phi_i \cdot \nabla_{w,d} \phi_j,
+@f}
+combining the coefficients we have calculated previous, it is calculated as
+@f{equation*}
+\int_E \mathbf{K} \sum_{k,l = 1}^{m}c_{ik} c_{jl} \left(\mathbf{w}_i \cdot \mathbf{w}_j\right)
+= \sum_{k,l = 1}^{m}c_{ik} c_{jl} \int_{E} \mathbf{K} \left(\mathbf{w}_i \cdot \mathbf{w}_j \right).
```

Comment:
Isn't it better to say `$\nabla_{w,d} \phi_i=\sum_k c_{ik} \mathbf{w}_k$` and `$\nabla_{w,d} \phi_j=\sum_k c_{jl} \mathbf{w}_l$`? `i`and `j`belong to `\phi_i` and `\phi_j` and not to `w_i`and `w_j`.

Reply:
Shouldn't this be (in the first line) 
`int_E \sum_{k,l = 1}^{m}c_{ik} c_{jl} \left(\mathbf{K} \mathbf{w}_i \cdot \mathbf{w}_j\right)`?
Otherwise, you are multiplying a matrix with a vector.